darts/tests/models/forecasting/test_torch_forecasting_model.py

```diff
@@ -78,6 +79,23 @@
 ]
 
 
+class NumsCalled(Metric):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
Not required for this test
```suggestion
    def __init__(self):
        super().__init__()
```

Reply:
removed